nemo_curator/classifiers/domain.py

```diff
@@ -49,30 +53,32 @@ def __init__(
         if max_mem_gb is None:
             max_mem_gb = _get_suggest_memory_for_classifier()
 
-        super().__init__(self.config.model, max_mem_gb=max_mem_gb)
+        super().__init__(self.config.base_model, max_mem_gb=max_mem_gb)
 
     def load_model(self, device: str = "cuda"):
-        model = HFDeberta.from_pretrained(DOMAIN_IDENTIFIER)
+        model = HFDeberta.from_pretrained(self.config.identifier)
         model.set_autocast(self.autocast)
         model = model.to(device)
         return model.eval()
 
     def load_tokenizer(self):
-        return AutoTokenizer.from_pretrained(DOMAIN_IDENTIFIER)
+        return AutoTokenizer.from_pretrained(self.config.identifier)
 
     def load_config(self):
-        return AutoConfig.from_pretrained(DOMAIN_IDENTIFIER)
+        return AutoConfig.from_pretrained(self.config.identifier)
 
 
 class DomainClassifier(DistributedDataClassifier):
     """
-    DomainClassifier is a specialized classifier designed for domain classification tasks, utilizing the
-    NVIDIA Domain Classifier model (https://huggingface.co/nvidia/domain-classifier). This class is optimized
-    for running on multi-node, multi-GPU setups to enable fast and efficient inference on large datasets.
+    DomainClassifier is a specialized classifier designed for domain classification tasks,
+    utilizing the NVIDIA Domain Classifier (https://huggingface.co/nvidia/domain-classifier)
+    and the NVIDIA Multilingual Domain Classifier (https://huggingface.co/nvidia/multilingual-domain-classifier) models.
+    This class is optimized for running on multi-node, multi-GPU setups to enable fast and efficient inference on large datasets.
 
     Attributes:
         filter_by (list[str], optional): The classes to filter the dataset by.
                                          If None, all classes will be included. Defaults to None.
+        multilingual (bool): If True, enable domain classification across 52 languages.
```

Comment:
It's worth calling out that this will use a separate model. Probably good to link to the model used in each case directly like you do in the overall docstring.

Reply:
Updated, thanks.